static/app/views/eventsV2/resultsChart.tsx

```diff
@@ -148,46 +155,67 @@ class ResultsChartContainer extends Component<ContainerProps> {
     } = this.props;
 
     const hasQueryFeature = organization.features.includes('discover-query');
-    const displayOptions = eventView.getDisplayOptions().filter(opt => {
-      // top5 modes are only available with larger packages in saas.
-      // We remove instead of disable here as showing tooltips in dropdown
-      // menus is clunky.
-      if (
-        [DisplayModes.TOP5, DisplayModes.DAILYTOP5].includes(opt.value as DisplayModes) &&
-        !hasQueryFeature
-      ) {
-        return false;
-      }
-      return true;
-    });
+    const hasConnectDiscoverAndDashboards = organization.features.includes(
+      'connect-discover-and-dashboards'
+    );
+    const displayOptions = eventView
```

Comment:
What are your thoughts on changing the behaviour so if `Top 5 Daily` was selected, and we select >1 yAxis that instead of defaulting back to `Total Period` we default to `Total Daily`?

Reply:
I like this! Updated `Top 5 Daily` to default back to `Total Daily` when using > 1 yAxis. Other display options will still default to `Total Period`.